tests/gflownet/envs/test_spacegroup.py

```diff
@@ -88,6 +109,42 @@ def test__action_space__contains_expected(env, action, expected):
     assert (action in env.action_space) == expected
 
 
+@pytest.mark.parametrize(
+    "action, expected",
+    [
+        (
+            (2, 1, 0),
+            True,
+        ),
+        (
+            (2, 17, 0),
+            True,
+        ),
+        (
+            (2, 39, 0),
+            True,
+        ),
+        (
+            (2, 123, 0),
+            True,
+        ),
+        (
+            (2, 230, 0),
+            True,
+        ),
+        (
+            (2, 2, 0),
+            False,
+        ),
+    ],
+)
+def test__action_space__env_with_restricted_spacegroups__contains_expected(
+    env_with_restricted_spacegroups, action, expected
+):
+    env = env_with_restricted_spacegroups
+    assert (action in env.action_space) == expected
```

Comment:
We should also test actions for CLS and PS that should be available (or not) based on the selected space groups.